### PR TITLE
ESC Content Enhancements

### PR DESCRIPTION
**Pull request by toriancrane**
## Description

Updated content location:
- `/docs/esc/get-started/`

This PR does the following:

- Moves existing ESC content from under Pulumi Cloud to standalone top level
- Removes Get Started long form tutorial
- Adds the following pages:
  - Get started hub page
  - Before you begin
  - Create environment
  - Store and retrieve secrets
  - Import environments
  - Run commands without local secrets

Resolves the following issues:

- https://github.com/pulumi/docs/issues/10602
- https://github.com/pulumi/docs/issues/10376
- https://github.com/pulumi/docs/issues/10377
- https://github.com/pulumi/docs/issues/10378
- https://github.com/pulumi/docs/issues/10382
- https://github.com/pulumi/docs/issues/10379

### CODE REVIEW

File: themes/default/content/docs/esc/get-started/import-environments.md
Comment (scottslowe) [inline themes/default/content/docs/esc/get-started/import-environments.md:1]: Is this file supposed to show up somewhere? I don't see it in the menu, and I can't seem to access it via URL, either.

**Review by scottslowe — CHANGES_REQUESTED** [review]
Generally looks great, but I found one file that appears to be orphaned (the page on importing environments). I wasn't sure if that was intentional, or accidental.

File: themes/default/content/docs/esc/get-started/import-environments.md
Comment (toriancrane) [reply themes/default/content/docs/esc/get-started/import-environments.md:1]: I think this had something to do with the merge conflict. It should be showing up now.

**Review by scottslowe — APPROVED** [review]

File: themes/default/content/docs/esc/_index.md [hunk 1]
```diff
@@ -3,22 +3,28 @@ title: Pulumi ESC
 title_tag: Pulumi ESC (Environments, Secrets, and Configuration)
 h1: Pulumi ESC (Environments, Secrets, and Configuration)
 meta_desc: Pulumi ESC allows you to compose and manage hierarchical collections of configuration and secrets and consume them in various ways.
+no_on_this_page: true
```
Comment (cnunciato) [inline themes/default/content/docs/esc/_index.md:6]: Removing this line will get the right-hand nav to show up:

```suggestion
```

**Pull request merged**